AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.ts

```diff
@@ -243,69 +240,82 @@ export class SummaryComponent implements OnInit, OnDestroy {
                     caseName: this.hearing.cases[0].name,
                     caseNumber: this.hearing.cases[0].number
                 });
+
                 const hearingDetailsResponse = await this.hearingService.saveHearing(this.hearing);
 
-                if (hearingDetailsResponse.status === BookingStatus.Failed.toString()) {
-                    this.hearing.hearing_id = hearingDetailsResponse.id;
-                    this.setError(`Failed to book new hearing for ${hearingDetailsResponse.created_by} `);
-                    return;
+                if (this.hasJudge(hearingDetailsResponse)) {
```

Comment:
if there's no judge then we still need to do the steps below (clone multi days, navigate to the booking confirmation page) but without polling for the status, which is specific to hearings with a judge.

probably best to refactor lines 252-259 into a separate method so they can also be called if there is no judge.

Reply:
done